com.unity.shadergraph/Documentation~/Texture-Size-Node.md

```diff
@@ -0,0 +1,42 @@
+# Texture Size Node
+
+The Texture Size node takes a Texture 2D input and returns the width and height texel resolution of the texture. It also returns the width and height size of each texel of the texture. The node uses the built in variable `{texturename}_TexelSize` to access the special properties of the given Texture 2D input.
+
+<!-- ![](images/) Add image of node-->
+
+If you experience texture sampling errors while using this node in a graph which includes Custom Function Nodes or Sub Graphs, you can resolve them by upgrading your version of Shader Graph to version 10.3 or later.
+
+> [!NOTE]
+> Don't use the default input to reference your **Texture 2D**, as this affects the performance of your graph. Connect a [Texture 2D Asset Node](Texture-2D-Asset-Node.md) to the Texture Size node's Texture input port and re-use this definition for sampling.
+
+## Create Node menu category
+
+The Texture Size node is under the **Input** &gt; **Texture** category in the Create Node menu.
+
+## Compatibility
+
+The Texture Size node is compatible with all render pipelines.
+
+## Ports
+
+| Name         | Direction | Type     | Binding | Description |
+|:------------ |:----------|:---------|:--------|:------------|
+| Texture      | Input     | Texture  | None    | The Texture 2D asset to measure. |
+| Width        | Output    | Float    | None    | The width of the Texture 2D asset in texels. |
+| Height       | Output    | Float    | None    | The height of the Texture 2D asset in texels.. |
```

Comment:
There's an extra period here :) Make sure you delete that!

Reply:
Done and done.